project-base/tests/App/Acceptance/acceptance/CartCest.php

```diff
@@ -146,7 +147,7 @@ public function testChangeCartItemAndRecalculatePrice(
         $me->amOnLocalizedRoute('front_product_detail', ['id' => 1]);
         $me->seeTranslationFrontend('Add to cart');
         $productDetailPage->addProductIntoCart(3);
-        $me->clickByTranslationFrontend('Go to cart');
+        $me->clickByTranslationFrontend('Go to cart', 'messages', [], WebDriverBy::cssSelector('#window-main-container'));
```

Comment:
I don't understand this, can you please explain to me the reason for this change?

Reply:
Yes, in `boxAction.html.twig` is new link to cart, exactly `<a href="{{ url('front_cart') }}" class="btn btn--success">{{ 'Go to cart'|trans }}</a>` and all tests find first `'Go to cart'` string on page (yes, tests find link in boxAction). But we want click to other `Go to cart link` - to link in `#window-main-container` parent element.

Reply:
thank you, I'm thinking about DRY. 
What do you think about create method in `OrderPage` (maybe CartPage too)
```php
public function clickMainGoToCart()
{
    $this->tester->clickByTranslationFrontend('Go to cart', 'messages', [], WebDriverBy::cssSelector('#window-main-container'));
}
```

and in tests only call `orderPage->clickMainGoToCart()`?
I believe it could help to understand the tests in the future.

Reply:
ok, fixed